packages/react-components/react-overflow/src/components/OverflowDivider/OverflowDivider.types.ts

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+
+/**
+ * OverflowDividerProps
+ */
+export type OverflowDividerProps = {
+  /**
+   * Assigns the item to a group, group visibility can be watched.
+   */
+  groupId?: string;
```

Comment:
```suggestion
  groupId: string;
```

if the groupId is not optional to register dividers, the groupId should not be optional in the component